### ISSUE TITLE
There should be a shipped example of a PNG8 setup.

### ISSUE THREAD

**Issue by GraionDilach** [OpenRA/OpenRA#15800]
Even the source code is unclear about how to set up the metadata properly, so the best would be to extract it from an already existing PNG, but there isn't such in the first-party.

The best suggestion I would make in this regard is to port the lighthouse from RA to TD, but letting it stay within RA palette, so that it can feature both usecases as an asset and as a palette.

**Comment by IceReaper** [comment]
Example png i use for my custom mousecursors:
![Cursors](https://www.kknd-game.com/assets/Cursors.png)

And the shared palette i am using for my whole kknd project:
![Palette](https://www.kknd-game.com/assets/palette.png)

Hope this helps as example :)

**Comment by reaperrr** [comment]
Added link to #15336 OP since that's where people arriving from the changelog entry will look first.

**Comment by GraionDilach** [comment]
The examples have been deleted.

**Comment by GraionDilach** [comment]
Please consider this part of Next+1 with shipping, say, the lighthouse in PNG8 format, to prevent cases like #17069.

**Comment by yaaaaa** [comment]
Yes, i use utility to make sequense.png file, but i have a questions abut palette and playercolor. instuction is needed...
1) playercolor colors must be same in al png files?
2) palette must be same in all png files?
